doc/chips/10.rst

```diff
@@ -460,50 +498,17 @@ their type when copies of it must be made, such as when the type is passed to a
 function or task by the ``in`` intent, or copied across locale boundaries.  An
 example of when such control would be desired is if the type designer wanted to
 implement reference counting for their type.  For these situations, the type
-designer may define a copy initializer, to handle the final state of the type's
-fields once a shallow bit copy has been made.
-
-Note that the cases handled by a copy initializer are not the same as user
-specified calls of the form:
+designer may define a copy initializer, to handle the state of the type's
+fields.  The copy initializer would be an initializer with a single argument, of
+the same type as the type being created.  The copy initializer would be called
+both in these cases and in user specified calls of the form:
```

Comment:
'these cases' refers to something too far away for me to remember. Maybe 'the cases listed above (such as with the in intent)'. But, we leave out what I view as the prime example:

  var a = b;

will copy-initialize a from b (ie use a's copy initializer).